src/blog/lagom-1-4-0-M2.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Lagom 1.4.0-M2!
+date: 2017-09-05
+author_github: ignasi35
+tags: news
+summary: >
+    Lagom 1.4.0-M2!
+---
+
+We are very pleased to announce the availability of the second Milestone release of Lagom 1.4.0. This is a preview release of the Lagom 1.4 series which is focused on upgrading all it's building blocks. This release includes Play 2.6 and Akka 2.4 like Lagom 1.4.0-M1 but both have been upgraded. It also includes the latest Slick 3.2.1 and Kafka 0.11.0.0. We are still adding the finishing touches to support Scala 2.12 in Lagom 1.4.x series.
+
+Phew, that was a mouthful.
+
+Unfortunately we've decided that [supporting sbt 1.0](https://github.com/lagom/lagom/issues/932) is not a feature we will work on before releasing Lagom 1.4.0. At the moment it still undecided if support for sbt 1.0 will come during the 1.4.x series or later.
+
+
+### Migrating from other versions
+
+Lagom 1.4.x series is not focused on providing new big features but, instead, it provides a collections of small improvements. We have included in the [change log](/changelog.html) a list of changes _exclusive_ to this release but that list is miss-leading:
```

Comment:
Maybe it would be better if we spent some time to create a change log that _isn't_ misleading and contains a complete list of changes from M1.